jipdate.py

```diff
@@ -191,7 +196,7 @@ def get_jira_issues(jira, exclude_stories, epics_only, all_status, filename,
         f.write("# Header: %s\n" % issue.fields.summary)
         f.write("# Type: %s\n" % issue.fields.issuetype)
         f.write("# Status: %s\n" % issue.fields.status)
-        f.write("No updates since last week.\n\n")
+        f.write("# No updates since last week.\n\n")
```

Comment:
I think that defeats the purpose, by removing that we fall back to not update everything on weekly basis. But, since there are different needs here I think we should move "extra comments" to a config file instead. I've done that here:
https://github.com/jbech-linaro/jipdate/commit/5c3f88f4c3651c27898f38b54ab7bba9657fa4f2